src/components/EventDetail.tsx

```diff
@@ -109,10 +109,21 @@ const EventDetail: FunctionComponent<CalendarUIProps> = ({
           )}
           {reservable && (
             <Button
-              variant="contained"
-              style={{ marginBottom: 30, alignSelf: "center" }}
+              key="MakeBooking"
+              style={{
+                backgroundColor: "Green",
+                color: "white",
+                maxWidth: "400px",
+              }}
+              onClick={(event): void => {
+                event.stopPropagation();
+                dispatch({
+                  type: CalendarAction.OpenReservationPage,
+                  payload: { currentEvent: state.currentEvent },
+                });
+              }}
             >
-              Book Me
+              Reserve Block
```

Comment:
Just a gut feeling that "block" may be unclear.  I could see some users hesitating if they didn't equate "block" to the event. 
```suggestion
              Reserve this time
```

Reply:
i like reserve this time. changed to that